layerInfo.json

```diff
@@ -99,7 +99,7 @@
 				],
 				"deps": ["Framework-Utils", "Loader", "Runtime", "Client-Utils"]
 			},
-			"ServiceClients": {
+			"TinyliciousClient": {
```

Comment:
How challenging is it to move TinyliciousClient into the service-clients directory too?  Not needed for this PR, but would take it if it's trivial.

Reply:
shouldn't be difficult I assume. I can have follow-up for this one too

Reply:
https://github.com/microsoft/FluidFramework/pull/19667